ckan/lib/task.py

```diff
@@ -0,0 +1,101 @@
+from rq import Queue
+from redis import Redis
+
+from logging import getLogger
+log = getLogger(__name__)
+
+redis_conn = Redis()
```

Comment:
Definitely don't think this should be setup like this (or here at all). Here's an example from one of my tools:

```python
if args['worker']:
        with app.app_context():
            with Connection(Redis.from_url(app.config['BROKER_URL'])):
                qs = [Queue(n) for n in args['--names']] or [Queue()]
                w = Worker(qs)
                w.work()
```

Just add this as a paster command.

Benefits:

1. Properly maintains the Connection stack
2. No connection state or details in tasks.py
3. Far more control over the queus than what you can get with the rqworker command (which is really just a dev tool)
4. Uses the config from Flask in this case, but would use the development.ini.